ci/cron/src/Main.hs

```diff
@@ -269,21 +270,13 @@ docs = do
 
 download_assets :: FilePath -> GitHubRelease -> IO ()
 download_assets tmp release = do
-    shell_ $ unlines ["bash -c '",
-        "set -euo pipefail",
-        "eval \"$(dev-env/bin/dade assist)\"",
-        "cd \"" <> tmp <> "\"",
-        "PIDS=\"\"",
-        "for ass in " <> unwords (map (show . uri) $ assets release) <> "; do",
-            "{",
-                "wget --quiet \"$ass\" &",
-            "}",
-            "PIDS=\"$PIDS $!\"",
-        "done",
-        "for pid in $PIDS; do",
-            "wait $pid",
-        "done",
-        "'"]
+    Control.Concurrent.Async.forConcurrently_ (map uri $ assets release) (\url -> do
```

Comment:
`forConcurrently_` will spawn an unbounded number of threads. I guess we don’t care since the number of assets is small? If we do care, the easiest solution is usually just to throw a semaphore around the map function you pass to `forConcurrently_`.

Reply:
The Bash code this replaces was also unbounded, so at least this is not getting worse. 🙂

I'm not concerned; we're not going to have a very big number of files anyway. If you are concerned, I can add a semaphore with, say, 30 slots, so it won't change anything for the happy path (16 assets atm) but will shield against future issues. I'd prefer to do that in a future PR, though.